tests/TestCase/Controller/Component/AuthComponentTest.php

```diff
@@ -23,6 +23,7 @@
 use Cake\Network\Request;
 use Cake\Network\Response;
 use Cake\ORM\TableRegistry;
+use Cake\Routing\Route\InflectedRoute;
```

Comment:
@antograssiot Do you think we can fix the sniffer/stickler to provide the expected order validation?